### ISSUE TITLE
Hover broken in 1.16.0-insider

### ISSUE THREAD

**Issue by stef-levesque** [stef-levesque/vscode-hexdump#31]
The hover message is now broken in the 1.16.0-insider version of VSCode. The 1.15.1 version is fine.

The hover content is defined in [extension.ts](/stef-levesque/vscode-hexdump/blob/3da7bef847a3a96249d5164c5e1c114de0546fac/src/extension.ts#L268) and the coloration from [hexdump.tmLanguage](https://github.com/stef-levesque/vscode-hexdump/blob/3da7bef847a3a96249d5164c5e1c114de0546fac/syntaxes/hexdump.tmLanguage)
```typescript
return new vscode.Hover( {language: 'hexdump', value: "content"} );
```

@jrieken @dbaeumer 
Could this come from Microsoft/vscode#29076 ?
Is it possible that Hover doesnt take extention .tmLanguage anymore ?
Anyway to reproduce the same colorization with a MarkdownString ?

![image](https://user-images.githubusercontent.com/7251102/29830265-ce46a6e2-8caf-11e7-80c3-8de8275cecad.png)

![image](https://user-images.githubusercontent.com/7251102/29830283-d74e0118-8caf-11e7-8367-4199ea74cc03.png)

**Comment by jrieken** [comment]
Thanks, I will investigate. We did refactor some bits but everything should still work...

**Comment by stef-levesque** [comment]
fwiw, it seems to always assume a MarkedString, or MarkdownString, whatever the language set.

For example, this display fine:
```typescript
content = '```hexdump\n' + content + '\n```\n';
return new vscode.Hover( {language: 'plaintext', value: content} );
```

**Comment by jrieken** [comment]
Yeah, it's is fixed on master. The problem is that the new `MarkdownString` has `value` and `isTrusted` and that this function also accepts a `MarkedString`

```ts
export function isMarkdownString(thing: any): thing is IMarkdownString {
	if (thing instanceof MarkdownString) {
		return true;
	} else if (typeof thing === 'object') {
		return typeof (<IMarkdownString>thing).value === 'string'
			&& (typeof (<IMarkdownString>thing).isTrusted === 'boolean' || (<IMarkdownString>thing).isTrusted === void 0);
	}
	return false;
}
```

**Comment by stef-levesque** [comment]
Now fixed in 1.16.0-insider
